proposed/2023/release-staging-and-deprecation.md

```diff
@@ -0,0 +1,191 @@
+- `failed` : operation failed
+When failed, the result of the command shuold also produce detailed information about what went wrong.
+
+### Technical explanation
+
+<!-- Explain the proposal in sufficient detail with implementation details, interaction models, and clarification of corner cases. -->
+
+## Drawbacks
+
+<!-- Why should we not do this? -->
+
+## Rationale and alternatives
+
+<!-- Why is this the best design compared to other designs? -->
+<!-- What other designs have been considered and why weren't they chosen? -->
+<!-- What is the impact of not doing this? -->
+
+### Alternatives considrered
+Utilizing **package tags** was considered as an alternative to solve the grouping problem.
+ Unfortunately, this has issues and can't be used for this purpose. Below are some of them:
+- There are currently no restrictions on tags usage, so anyone can use the same tag for a random package and "interfere" with a set
+- tags were introduced mostly for description and discovery purposes and are expected to be
+ hints on their own, so customers can utilise them in their searches. If a package author decides to use a tag to identify a release,
+ they may end up using a GUID as a tag, and that will introduce some user-friendliness related issues on NuGet.
+
+## Prior Art
+
+<!-- What prior art, both good and bad are related to this proposal? -->
+<!-- Do other features exist in other ecosystems and what experience have their community had? -->
+<!-- What lessons from other communities can we learn from? -->
+<!-- Are there any resources that are relevant to this proposal? -->
+
+Below are two problems from Microsoft, where due to lack of this feature teams are struggling and have to come up with potential alternative solutions.
+
+### .NET Release publishing
+
+The .NET team has a monthly release cadance for their products. As part of each release we publish hundreds of packages to NuGet.
+This is generally a very time-sensitive effort, especially when it comes to security releases. Many teams within Microsoft
+ wait for our releases before they can move forward with publishing their patches immediately after, because with security releases
+ we will also document the security fixes the release contains. Unfortunately, what happens sometimes, is that
+ during publishing NuGet sometimes blocks certain packages due to some new validation rules or some other reason, which
+ wasn't something the team could have foreseen earlier. As a result, the the releases get delayed and partner teams delay
+ their releases awaiting a green light from the .NET team, which results in a tense / stressfull effort of patching the impacting
+ packages on the last minute to address the faced issues.
+Besides the security releases, there are also GA releases, which have a similar risk characteristics.
+ Many times there are confrerences schedueld the same day when a release is planned. And during such a conference the presenter
```

Comment:
```suggestion
Many times there are conferences scheduled the same day when a release is planned. And during such a conference the presenter
 expectes the release to be published (a few hours earlier) so that they can demo some new feature(s).
 And it happened in the past that the releases didn't succeed in time and impacted the presentations in a negative way.
```